content/en/docs/1.get-started/1.installation.md

```diff
@@ -122,7 +122,7 @@ npm install nuxt
 ```
 ::
 
-This command will add `nuxt` as a dependency to your project and add it to your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
+This command will add `nuxt` as a dependency to your project and include it in your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
```

Comment:
I think the original was clearer here.
```suggestion
This command will add `nuxt` as a dependency to your project and add it to your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
```

Reply:
I think "include" is the more appropriate word but okay.